### PR TITLE
added libelemental

### PR DESCRIPTION
**Pull request by gidiko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,17 @@
 ## About
```
Comment (synapticarbors) [inline README.md]: Please do not edit this file

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -1,70 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not delete this file.

File: recipes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package:
```
Comment (synapticarbors) [inline recipes/meta.yaml]: Please move this recipe into `recipes/libelemental/meta.yaml`

File: recipes/build.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```
Comment (synapticarbors) [inline recipes/build.sh]: Please move this file to `recipes/libelemental/build.sh`

**Comment by mrocklin** [comment]
I would like to see a libelemental recipe on conda-forge.  I notice that @scizo has a package on anaconda.org.  Pinging him in case he has the time to revitalize this effort.

**Comment by mrocklin** [comment]
Also pinging @poulson

**Pull request closed without merging**